duckduckgo/iOS
# Adds the build number to the set_version.sh script

Task/Issue URL: https://app.asana.com/0/1200194497630846/1203547370781439/f

**Description**:

Introduces some changes to `set_version.sh` so that the build number is updated as well.  I initially contemplated using different scripts for setting the version and build number but I believe the build number is important enough that we want to make sure it's properly set whenever the version number is changed.

**Steps to test this PR**:
1. Run `set_version.sh` and check that the usage description looks correct.  Both the version number and build number should be properly reported below the usage instructions (from the file at `Configuration/Version.xcconfig`).
2. Run `set_version.sh 7.78 1` and ensure the version and bulid number are properly reflected in the project file for the following targets:
    - DuckDuckGo
    - ShareExtension
    - OpenAction
    - WidgetsExtension
3. Make sure that the version is also correctly changed in `DuckDuckGo/Settings.bundle/Root.plist b/DuckDuckGo/Settings.bundle/Root.plist`. 

---
###### Internal references:
[Software Engineering Expectations](https://app.asana.com/0/59792373528535/199064865822552)
[Technical Design Template](https://app.asana.com/0/59792373528535/184709971311943)
**When ready for review, remember to post the PR in MM**
